lib/builtins/deploy-delegates/lambda-deployer/helper.js

```diff
@@ -172,9 +172,13 @@ function _createLambdaFunction(reporter, lambdaClient, options, callback) {
                     return callback(revisionErr);
                 }
                 callback(null, {
-                    arn,
-                    lastModified: lambdaData.LastModified,
-                    revisionId: revisionData.Configuration.RevisionId
+                    isAllStepSuccess: true,
+                    isCodeDeployed: true,
+                    lambdaResponse: {
+                        arn,
+                        lastModified: lambdaData.LastModified,
+                        revisionId: revisionData.Configuration.RevisionId
```

Comment:
very minor:
here you have:
lastModified
revisionId

another place:
revisionId
lastModified

they properties switch order in ask-states.json depending if deploy failed or not. probably should stay consistent.